### ISSUE TITLE
translate chinese > english?

### ISSUE THREAD

**Issue by dcsan** [dushaobindoudou/vscode-translation#8]
would it be possible to make this work the other way around?

It's very useful that it doesn't use the google API, since I'm based in China.

**Comment by dushaobindoudou** [comment]
If I update next time, I will add support for this.

**Comment by 21307369** [comment]
https://translate.google.cn/

**Comment by dcsan** [comment]
@21307369 yes it seems the web service is one of the few not blocked google services.
but google cloud APIs do not work here.
